oci/oci.go

```diff
@@ -486,6 +486,9 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 	defer c.opLock.Unlock()
 	out, err := exec.Command(r.Path(c), "state", c.name).CombinedOutput()
 	if err != nil {
+		if strings.Contains(string(out), "does not exist") {
```

Comment:
Same here: This is `runc` specific and I think we don't want that.
Same remark as before: Could we just ping the process when `exec.Command()` fails and then return an ocid, named error that we can treat specifically ?